pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -831,14 +815,9 @@ public List<String> getAllRawTables() {
    * @return List of raw table names in provided database name
    */
   public List<String> getAllRawTables(@Nullable String databaseName) {
-    Set<String> rawTableNames = new HashSet<>();
-    for (String resourceName : getAllResources()) {
-      if (TableNameBuilder.isTableResource(resourceName)
-          && DatabaseUtils.isPartOfDatabase(resourceName, databaseName)) {
-        rawTableNames.add(TableNameBuilder.extractRawTableName(resourceName));
-      }
-    }
-    return new ArrayList<>(rawTableNames);
+    return getAllResources().stream().filter(
+        resourceName -> TableNameBuilder.isTableResource(resourceName) && DatabaseUtils.isPartOfDatabase(resourceName,
+            databaseName)).map(TableNameBuilder::extractRawTableName).distinct().collect(Collectors.toList());
```

Comment:
nit: could you please help me understand why we need to call `distinct()`?

Reply:
E.g. `myTable_OFFLINE` and `myTable_REALTIME` should result in one raw table name `myTable`